content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -0,0 +1,61 @@
+- ***Added*** experimental [mesh-wide tracing configuration API](/docs/tasks/observability/distributed-tracing/configurability/). This API provides control of trace sampling rates, the [maximum tag lengths](https://github.com/istio/istio/issues/14563) for URL tags, and [custom tags extraction](https://github.com/istio/istio/issues/13018) for all traces within the mesh.
+- ***Improved*** Prometheus integration experience by adding standard Prometheus scrape annotations to proxies and the control plane workloads. This removes the need for specialized configuration to discover and consume Istio metrics. More details are available in the [operations guide for Prometheus](/docs/ops/integrations/prometheus#option-2-metrics-merging/).
+- ***Improved*** support for [customizing v2 metrics generation](/docs/tasks/observability/metrics/customize-metrics/). This allows mesh operators to add and remove labels used in Istio metrics based on expressions over the set of available request and response attributes.
+- ***Updated*** default telemetry v2 configuration to avoid using host header to extract destination service name at gateway. This prevents unbound cardinality due to untrusted host header and implies that destination service labels are going to be omitted for request hits `blackhole` and `passthrough` at gateway.
+- ***Added*** automated publishing of Grafana dashboards to `grafana.com` as part of the Istio release process. Please see the [Istio org page](https://grafana.com/orgs/istio) for more information.
+- ***Updated*** Grafana dashboards to adapt to the new Istiod deployment model.
+
+## Configuration Management
+
+## Installation
+
+- ***Improved*** Please see the [Upgrade guide](/docs/setup/upgrade/) for migration. In-place upgrades are now possible in Istio.
+- ***Removed*** the legacy Helm charts.
+- ***Added*** Allow users to add a custom hostname for istiod.
+- ***Changed*** Gateway ports used (15020) and [resolution](https://github.com/istio/istio/pull/23432#issuecomment-622208734) for end users.
+- ***Added*** functionality to save installation state in a `CustomResource` in the cluster.
+- ***Improved*** installation to not manage the installation namespace, allowing more flexibility.
+- ***Removed*** the Citadel, Sidecar Injector, and Galley deployments. These were disabled by default in 1.5, and all functionality has moved into Istiod.
+- ***Removed*** the legacy `istio-pilot` configurations, such as Service.
+- ***Removed*** ports 15029-15032 from the default `ingressgateway`. It is recommended to expose telemetry addons by [host routing](/docs/tasks/observability/gateways/) instead.
+- ***Removed*** built in Istio configurations from the installation, including the Gateway, `VirtualServices`, and mTLS settings.
+- ***Added*** a new preview profile, allowing users to try out new experimental features that include WASM enabled telemetry v2.
```

Comment:
Can you say the new profile name here just for clarity?

Reply:
It's called `preview`. I'll try to make sure that is more explicit.